nosql/1.0/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Jakarta NoSQL"
```

Comment:
Add 1.0 here, so the title is "Jakarta NoSQL 1.0"

Reply:
Hey @ivargrimstad that is not the final version, it is the draft one.

Reply:
@otaviojava We are good to go as soon as you change this line to

title: "Jakarta NoSQL 1.0 (under development)"

Check out other specifications to see how they have done it.

Reply:
Thank you @ivargrimstad Done